backend/src/main/resources/application.yaml

```diff
@@ -4,14 +4,21 @@ spring:
   main:
     banner-mode: "OFF"
   datasource:
-    username: simple_report_app
-    password: api123
-    url: jdbc:postgresql://localhost:${SR_DB_PORT:5432}/simple_report
-    hikari:
-      maximum-pool-size: 50 # Maximum size that the pool is allowed to reach, including both idle and active connections. Effectively reflects number of concurrent database request threads. (Ex: A size of 50 represents a maximum capacity of 50 concurrent DB requests.)
-      max-lifetime: 600000 # Maximum lifetime for a connection to be retained in the pool, in milliseconds, once it is closed.
-      minimum-idle: 30 # Minimum number of idle connections to be retained in the pool. When a connection has been idle for the length of time specified in `idle-timeout`, it will be purged from the pool. This will allow for any un-closed, inactive transactions to be cleaned up more readily.
-      # idle-timeout: 600000 # Maximum lifetime for idle connections before they are purged from the pool. Only active if `minimum-idle` is set to less than `maximum-pool-size`. Defaults to 10 minutes.
+    simplereport:
+      username: simple_report_app
+      password: api123
+      hikari:
+        jdbc-url: jdbc:postgresql://localhost:${SR_DB_PORT:5432}/simple_report
+        maximum-pool-size: 50 # Maximum size that the pool is allowed to reach, including both idle and active connections. Effectively reflects number of concurrent database request threads. (Ex: A size of 50 represents a maximum capacity of 50 concurrent DB requests.)
+        max-lifetime: 600000 # Maximum lifetime for a connection to be retained in the pool, in milliseconds, once it is closed.
+        minimum-idle: 30 # Minimum number of idle connections to be retained in the pool. When a connection has been idle for the length of time specified in `idle-timeout`, it will be purged from the pool. This will allow for any un-closed, inactive transactions to be cleaned up more readily.
+        # idle-timeout: 600000 # Maximum lifetime for idle connections before they are purged from the pool. Only active if `minimum-idle` is set to less than `maximum-pool-size`. Defaults to 10 minutes.
+    metabase:
+      username: simple_report_app
+      password: api123
+      hikari:
+        maximum-pool-size: 2
```

Comment:
Do we also want to set the max lifetime and minimum idle properties on the metabase data source? I seem to remember it was greedy with our database connections, so I'm not sure if we can limit it more aggressively now that the data sources have been split.

Reply:
I don't think we need to if we limit the max pool size. This is confusing, but this is the pool size settings for the Liquibase migration for Metabase, not Metabase itself (which then runs its _own_ Liquibase migration that it manages). Metabase itself needs to have its pool limited via the app configuration (JVM setting, etc.).

Reply:
Aha, thank you! I hadn't realized this was just the pool settings size for the migration, makes sense.